clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  Button,
+  Heading,
+  HStack,
+  Link,
+  Stack,
+} from "@fidesui/react";
+import { Form, Formik } from "formik";
+import React from "react";
+import * as Yup from "yup";
+
+import { CustomSelect, CustomTextInput } from "../common/form/inputs";
+import {
+  AWS_REGION_OPTIONS,
+  DOCS_URL_AWS_PERMISSIONS,
+  DOCS_URL_IAM_POLICY,
+} from "./constants";
+import { useGenerateMutation } from "./scanner.slice";
+import { ScannerGenerateResponse } from "./types";
+
+// TODO: There should just be a theme/variant for blue links.
+const DocsLink = (props: React.ComponentProps<typeof Link>) => (
+  <Link isExternal color="complimentary.500" {...props} />
+);
+
+const initialValues = {
+  aws_access_key_id: "",
+  aws_secret_access_key: "",
+  region_name: "",
+};
+
+type FormValues = typeof initialValues;
+
+const ValidationSchema = Yup.object().shape({
+  aws_access_key_id: Yup.string().required().label("Access Key ID"),
+  aws_secret_access_key: Yup.string().required().label("Secret"),
+  region_name: Yup.string().required().label("Default Region"),
+});
+
+const AuthenticateAwsForm = () => {
+  // TODO: The organization key needs to come from the shared wizard state.
```

Comment:
I need to reach out about how we want to track the data that flows through the wizard. I'm thinking we can create a slice for the wizard and each form can hook in to update the relevant values. The `step` variables deserve to live there as well, so that we don't have to pass number-incrementing-callback props a bunch.

Reply:
makes sense to me! @LKCSmith would be the one to coordinate with on that